extensions/guacamole-auth-radius/pom.xml

```diff
@@ -150,14 +146,15 @@
             <version>1.1.5</version>
             <exclusions>
 
-                <exclusion>
-                    <groupId>org.slf4j</groupId>
-                    <artifactId>slf4j-api</artifactId>
-                </exclusion>
```

Comment:
I don't think you're going to be able to remove this - I think this causes the error I commented about with the conflict in SLF4J.

Reply:
Ah, you're completely right. SLF4J is being pulled into the extension, causing an identity crisis with the version in the webapp.